app-text/ripgrep/ripgrep-0.10.0.recipe

```diff
@@ -0,0 +1,62 @@
+SUMMARY="Combines the usability of ag with the raw speed of grep"
+DESCRIPTION="ripgrep is a line-oriented search tool that recursively searches \
+your current directory for a regex pattern while respecting your gitignore \
+rules. It is similar to other popular search tools like The Silver Searcher, \
+ack and grep."
+HOMEPAGE="https://github.com/BurntSushi/ripgrep"
+COPYRIGHT="2015 Andrew Gallant"
+LICENSE="MIT
+	Public Domain"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="a2a6eb7d33d75e64613c158e1ae450899b437e37f1bfbd54f713b011cd8cc31e"
+SOURCE_FILENAME="ripgrep-$portVersion.tar.gz"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+fi
+
+PROVIDES="
+	ripgrep$commandSuffix = $portVersion
+	cmd:rg
+	cmd:rg$commandSuffix
+	"
+REQUIRES="
+	haiku$commandSuffix
+	gcc${commandSuffix}_syslibs
+	"
+
+BUILD_REQUIRES="
+	haiku${commandSuffix}_devel
+	lib:libxslt$commandSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cargo$commandSuffix
+	cmd:gcc$commandSuffix
+	cmd:a2x
+	"
+
+BUILD()
+{
+	cargo build --release
+}
+
+INSTALL()
+{
+	install -Dm555 target/release/rg $binDir/rg
```

Comment:
I using $commandSuffix will work, you should use $prefix/bin here instead of $binDir, and then no ln -s would be required